### DISCUSSION TITLE
Access full_url of the uploaded images

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2883] [Archive]
I'm using S3 to store uploaded files. When I try to access `full_url` field I get the error:

```
{"error":{"code":1,"message":"Not Found"}}
```

Anything special I need to do to have these files accessible via the directus hosting link? Or I need to use the S3 URL for that? 

I see files are uploaded OK to S3 and I can access them from S3. 

Thx.

**Comment by TorbenHammes** [comment]
I had the same problem. I use local storage, but I can access the files via the API with the endpoint `/assets/<file-id>`. If you need the file publicly (without token) you can do that by enabling the role Public for the System Collection `Directus Files`.

Thanks to @rijkvanzanten for your help before!

I have problems connecting to the S3 Bucket. What does your .env file look like? Maybe you can help me here. Thank you :-D

**Comment by rijkvanzanten** [comment]
What `full_url` field?

The Directus API doesn't return a `full_url` field in `/files`
